.github/workflows/build.yml

```diff
@@ -438,43 +378,48 @@ jobs:
           # fix project path in NSIS script
           sed -i -e 's/c\:\\projects/d:\\a\\qlcplus/g' platforms/windows/${{env.NSIS_SCRIPT}}
 
-      - name: Fix 32 bit build
-        if: ${{ matrix.task == 'compile-qt5-32bit' }}
-        shell: msys2 {0}
-        run: |
-          # fix system libs path
-          sed -i -e 's/mingw64/mingw32/g' platforms/windows/CMakeLists.txt
-          # fix gcc lib
-          sed -i -e 's/libgcc_s_seh/libgcc_s_dw2/g' platforms/windows/CMakeLists.txt
-          # fix DMX USB path
-          sed -i -e 's/amd64/i386/g' plugins/dmxusb/src/CMakeLists.txt
-          sed -i -e 's/ftd2xx64.dll/ftd2xx.dll/g' plugins/dmxusb/src/CMakeLists.txt
-
       - name: Configure v4 build for Windows
         shell: msys2 {0}
-        if: ${{ matrix.task == 'compile-qt5' || matrix.task == 'compile-qt5-32bit' }}
+        if: ${{ matrix.task == 'compile-v4' }}
         run: |
-          cmake -S . -B build -G "Unix Makefiles" -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+          cmake -S . -B build -DCMAKE_PREFIX_PATH="${{ env.QTDIR }}/lib/cmake"
 
       - name: Configure v5 build for Windows
         shell: msys2 {0}
-        if: ${{ matrix.task == 'compile-qt5qml' }}
+        if: ${{ matrix.task == 'compile-v5' }}
         run: |
-          cmake -S . -B build -G "Unix Makefiles" -Dqmlui=ON -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+          cmake -S . -B build -DCMAKE_PREFIX_PATH="${{ env.QTDIR }}/lib/cmake" -Dqmlui=ON
```

Comment:
did you forget to reenable ccache? for ninja builds you might need to add
if(CMAKE_CXX_COMPILER_LAUNCHER STREQUAL "ccache")
		set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
		set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
endif()